app/views/unattended/provisioning_templates/snippet/built.erb

```diff
@@ -35,9 +39,9 @@ EOF
 -%>
 <% end -%>
 if [ -x /usr/bin/curl ]; then
-  /usr/bin/curl -o /dev/null --noproxy \* <%= curl_opts.join(' ') %> --silent '<%= url %>'
+  /usr/bin/curl -o /dev/null <%= curl_opts.join(' ') %> --silent '<%= url %>'
 elif [ -x /usr/bin/wget ]; then
-  /usr/bin/wget -q -O /dev/null --no-proxy --method <%= method %> <%= wget_opts.join(' ') %> '<%= url %>'
+  /usr/bin/wget -q -O /dev/null --method <%= method %> <%= wget_opts.join(' ') %> '<%= url %>'
 else
   wget -q -O /dev/null --header 'Content-Type: text/plain' '<%= url %>'
```

Comment:
I assume this is something like busybox wget? So something like wget, but not exactly wget. Do we know what will happen in this case? Will it honor the system-wide proxy?

Reply:
Not sure, I could not find a good documentation for it. It seems you can pass '-Y/--proxy yes/no' option to busybox wget to use the proxy. If you can turn it off, I would assume that you can configure a default proxy or system proxy.
If this is the right code, it probably does https://coral.googlesource.com/busybox/+/refs/heads/release-chef/networking/wget.c#234